### PR TITLE
Adding StackNavigator with 2 complete screen - Splash and Mobile Logi…

### PR DESCRIPTION
**Pull request by codeshaker**
Adding StackNavigator with 2 complete screen - Splash and Mobile Login Screen

### CODE REVIEW

File: FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
```
Comment (harman8489) [inline FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js]: just import React from 'react' will work

File: FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js [hunk 2]
```diff
@@ -10,23 +11,62 @@ import {
 } from 'react-native';
 
 export default class MobileNumberForm extends Component {
```
Comment (harman8489) [inline FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js]: React.Component

File: FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js (hunk 1 above)
Comment (codeshaker) [reply FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js]: In code i am inheriting class MobileNumberForm from Component class so will need it.

File: FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js (hunk 2 above)
Comment (codeshaker) [reply FrontEnd/HomeClubUI/app/Components/MobileNumberForm.js]: Sorry i did not get your comment . please elaborate.

**Pull request merged**